test/endToEnd/authorization/usersTest.js

```diff
@@ -689,7 +683,6 @@ lab.experiment('Routes Authorizations', () => {
           Action: ['authorization:users:policy:remove'],
           Resource: ['/authorization/user/WONKA/{{calledTeam.id}}/*']
         }])
-        .skip()
         .shouldRespond(204)
```

Comment:
Can we add a test for a user that is not in any team? (maybe we already have some in the fixtures)

ie: 

user: `filippo` it's only in `WONKA`
resource to change it: `/authorization/user/WONKA/*/filippo`